api/insight/converter.go

```diff
@@ -150,19 +148,17 @@ func (c *insightApiContext) TxConverterWithParams(txs []*dcrjson.TxRawResult, no
 	return newTxs, nil
 }
 
-// BlockConverter converts dcrd-block to Insight block
-func (c *insightApiContext) BlockConverter(inBlocks []*dcrjson.GetBlockVerboseResult) ([]*apitypes.InsightBlockResult, error) {
-	params := &chaincfg.MainNetParams
```

Comment:
Pretty sure this was a bug.  I was about to remove the receiver (`(c *insightApiContext)`) from the function signature since `c` wasn't used, but I was the actual chaincfg params were needed.

Reply:
I think you are right.